src/wp-cron.php

```diff
@@ -18,19 +18,11 @@
 
 ignore_user_abort( true );
 
-if ( ! headers_sent() ) {
-	header( 'Expires: Wed, 11 Jan 1984 05:00:00 GMT' );
-	header( 'Cache-Control: no-cache, must-revalidate, max-age=0' );
```

Comment:
Can't these headers be left right here? They're now sent in two different places below.

Reply:
There's a slight change: if requests are bypassed the no-caching headers aren't sent as it's safe to cache the endpoint as it doesn't need to hit the database.

Reply:
Ah, right. Good point